microsoft-integration/admin-tool/src/main/resources/Messages_es.properties

```diff
@@ -1,183 +1,185 @@
-# Common messages 
-common_danger=You must be an admin to use this tool, please contact your Administrator.
-
-# Menu messages
-menu_main = Admin
-menu_new = New
-menu_new.title= Create a new site synchronization
-menu_edit = Edit
-menu_edit.title = Edit a site synchronization
-menu_config = Config
-menu_config.title = Edit microsoft configuration
-menu_credentials = Credentials
-menu_credentials.title = Microsoft Credentials
-
-#date format
-date.format=dd/MM/yyyy H\:m
-
-#Generic site synchronization messages
-site=Site
-site.id=Site Id: {0}
-group=Group
-group.id=Group Id: {0}
-team=Team
-team.partially_created=This Team is partially created
-team.id=Team Id: {0}
-channel=Channel
-channel.id=Channel Id: {0}
-status=Status
-status.OK=OK
-status.KO=KO
-status.NONE=None
-status.ERROR=Error
-status.PARTIAL_OK=Partial OK
-status.PARTIAL_KO=Partial KO
-status.ERROR_GUEST=Error Guest
-empty_table = No active synchronizations
-forced=Forced
-save=Save
-cancel=Cancel
-close=Close
-search=Search
-delete=Delete
-back=Back
-confirm=Confirm
-refresh=Refresh
-start=Start
-new=NEW
-found=FOUND
+# Mensajes comunes
+common_danger = Debes ser Administrador para usar esta herramienta, por favor contacta con tu Administrador.
+
+# Mensajes del menú 
+menu_main = Administrador
+menu_new = Nuevo
+menu_new.title = Crear nueva sincronizaci/00F3n del sitio
```

Comment:
This should be \u00F3, fix all the entities.

Reply:
Done!